fab/environments.yml

```diff
@@ -58,7 +58,7 @@ softlayer:
 
 icds:
   environment: icds
-  http_proxy: 169.38.84.79:3128
+  http_proxy: 10.247.63.132:3128
```

Comment:
Is it not easy to add this to inventory file, so that ansible and fab can pull from same place?